.travis.yml

```diff
@@ -5,6 +5,7 @@ php:
   - 7.1
   - 7.2
   - 7.3
+  - 7.4snapshot
```

Comment:
As PHP 7.4 is not release yet, I would make it an allowed failure for now.

What do you think ?

Reply:
Allowed failure or not, could you please merge this one?
This will be useful in order to work on 7.4 compatibility... Thanks!